src/Units/DoRUnits.java

```diff
@@ -672,6 +672,7 @@ public GunboatModel()
       baseCargoCapacity = 1;
       carryableMask = TROOP;
       carryableExclusionMask = TANK; // Can't carry Bikes
+      unloadExclusionTerrain.add(TerrainType.BRIDGE);
```

Comment:
So only DoR units are unable to unload on bridges? Your dedication to accurately recreating the nuances of cart differences is impressive.

Reply:
Trilogy Landers can't get on bridges. I think they also couldn't if you hacked them onto the bridge, but I only care about implementation details like that if they tickle my fancy.
I do think I probably want to make DoR-style bridges a separate terrain type eventually, but "We don't have a new terrain type" is probably not a bug.
(And they will totally be called `DRAWBRIDGE`s)

And thanks, I think.